crates/sui-e2e-tests/tests/zklogin_tests.rs

```diff
@@ -86,132 +85,119 @@ async fn test_legacy_zklogin_address_accept() {
+#[sim_test]
+async fn test_expired_zklogin_sig() {
+    let test_cluster = TestClusterBuilder::new()
+        .with_epoch_duration_ms(10000)
+        .with_default_jwks()
+        .build()
+        .await;
+
+    // trigger reconfiguration that advanced epoch to 1.
+    test_cluster.trigger_reconfiguration().await;
+
+    // trigger reconfiguration that advanced epoch to 2.
+    test_cluster.trigger_reconfiguration().await;
+    // load one test vector, the zklogin inputs corresponds to max_epoch = 1
+    let (kp, pk_zklogin, inputs) =
         &load_test_vectors("../sui-types/src/unit_tests/zklogin_test_vectors.json")[1];
     let zklogin_addr = (pk_zklogin).into();
+
+    let rgp = test_cluster.get_reference_gas_price().await;
     let gas = test_cluster
         .fund_address_and_return_gas(rgp, Some(20000000000), zklogin_addr)
         .await;
     let tx_data = TestTransactionBuilder::new(zklogin_addr, gas, rgp)
         .transfer_sui(None, SuiAddress::ZERO)
         .build();
-    let intent_msg = IntentMessage::new(Intent::sui_transaction(), tx_data.clone());
 
-    let sig: GenericSignature = ZkLoginAuthenticator::new(
-        zklogin_inputs.clone(),
-        10,
-        Signature::new_secure(&intent_msg, eph_kp),
-    )
-    .into();
-    let tx = Transaction::from_generic_sig_data(tx_data.clone(), vec![sig]);
+    let msg = IntentMessage::new(Intent::sui_transaction(), tx_data.clone());
+    let eph_sig = Signature::new_secure(&msg, kp);
 
-    let res = context.execute_transaction_may_fail(tx).await;
+    // combine ephemeral sig with zklogin inputs.
+    let generic_sig = GenericSignature::ZkLoginAuthenticator(ZkLoginAuthenticator::new(
+        inputs.clone(),
+        1,
+        eph_sig.clone(),
+    ));
+    let signed_txn_expired = Transaction::from_generic_sig_data(tx_data.clone(), vec![generic_sig]);
+    let context = &test_cluster.wallet;
+
+    let res = context
+        .execute_transaction_may_fail(signed_txn_expired)
+        .await;
     assert!(res
         .unwrap_err()
         .to_string()
-        .contains("ZKLogin expired at epoch 10"));
+        .contains("ZKLogin expired at epoch 1"));
 }
 
 #[sim_test]
-async fn zklogin_end_to_end_test_with_auth_state_creation() {
+async fn test_auth_state_creation() {
     // Create test cluster without auth state object in genesis
     let test_cluster = TestClusterBuilder::new()
         .with_protocol_version(23.into())
-        .with_epoch_duration_ms(10000)
+        .with_epoch_duration_ms(100)
         .with_default_jwks()
         .build()
         .await;
 
     // Wait until we are in an epoch that has zklogin enabled, but the auth state object is not
     // created yet.
     test_cluster.wait_for_protocol_version(24.into()).await;
-
-    // Now wait until the next epoch, when the auth state object is created.
-    test_cluster.wait_for_epoch(None).await;
-
-    // run zklogin end to end test
-    run_zklogin_end_to_end_test(test_cluster).await;
```

Comment:
this is removed bc its covered by `zklogin_end_to_end_test` hence use a small epoch duration and just check for jwk update.